technical-resources/loan-managers/open-term-loan-manager/open-term-loan-manager.md

```diff
@@ -0,0 +1,249 @@
+
+# Refinance
+- The PoolDelegate can call `proposeNewTerms()` to the Loan which will call the Loan's `proposeNewTerms()` and store the `refinanceCommitment` which is a hash of `keccak256(abi.encode(refinancer_, deadline_, calls_))`. 
+- The PoolDelegate can call `rejectNewTerms()` to the Loan which will call the Loan's `rejectNewTerms()` which will delete the `refinanceCommitment` not allowing the borrower to accept the new terms via the Loan.
+- When the borrower accepts new terms as part of the refinance flow the LoanManager's `claim()` is called as the borrower pays any interest and fees due. Importantly this flow also accounts for a change in `principal` where the principal can both increase in which case funds are requested from the Pool and sent to the borrower via the Loan, or if the principal is decreased the funds are sent back to the Pool. 
+- In the `claim()` function if the principal passed is positive that means that amount of principal is being repaid otherwise if negative that means that amount of principal is being requested from the Pool as part of a Refinance to increase the `principal`. 
+
+# Call / Remove Call
+- The PoolDelegate directly `callPrincipal()` with the principal amount, this then calls the Loan's `callPrincipal()`. No state changes occur on the LoanManager. 
+- The same is true for `removeCall()` which just calls the Loan's `removeCall()`. No state changes occur on the LoanManager
+
+# Impairment / Remove Impairment 
+- Both the PoolDelegate and Governor can call `impairLoan()` which calls the Loan's `impair()` and updates the LoanManager accounting by reducing the issuanceRate, updates the `unrealizedLosses` with the principal and accrued interest and stores the impairment information in the `Impairment` struct for future use. 
+- Both the PoolDelegate and Governor can call `removeLoanImpairment()` but if the Loan was Impaired by the Governor only the Governor can call `removeLoanImpairment()`, this is to safeguard against any malicious use. In the case `removeLoanImpairment()` is called the LoanManager accounting is updated to include the issuanceRate for the Loan payment and add back any Interest that was expected to be accrued during this period between the Impairment date and the current timestamp. Finally `removeImpairment()` is called on the Loan. 
+
+# Default
+- The PoolManager can call `triggerDefault()` on the LoanManager. This is due to the PoolManager also needing to handle PoolCover so the decision was made to keep the entry point to `triggerDefault()` the PoolManager. 
+- Note the interface for `triggerDefault(address loan_, address liquidatorFactory_)` takes in a liquidatorFactory address this is ignored for the execution in the Open-Term LoanManager and is in the interface for compatibility of the PoolManager with both Fixed/Open-Term LoanManagers. 
```

Comment:
```suggestion
- Note the interface for `triggerDefault(address loan_, address liquidatorFactory_)` takes in a `liquidatorFactory` address, which is ignored for the execution in the Open-Term LoanManager (since Open-Term Loans have no collateral) and is in the interface for compatibility of the PoolManager with both Fixed/Open-Term LoanManagers. 
```